breathecode/authenticate/tests/urls/tests_github_reset_link.py

```diff
@@ -0,0 +1,76 @@
+from breathecode.authenticate.models import ProfileAcademy, Token
```

Comment:
This file name is not using the correct name convention

Reply:
Claro, tu estas usando este nombre como el nombre de archivo porque lo colocastes como nombre del url equivocado, osea que no tiene nada que ver con el url original, este nombre de archivo no me aporta casi ninguna informacion sobre donde estoy parado, solo aporta que esta relacionado a resetear urls de github, y por ejemplo en el modulo de admissions donde hay muchas formas de listar una cohorte, colocar como opcion un test que se llame listar cohorte generaria ambiguedad entre todos los endpoints similares